src/test/java/com/squareup/javapoet/TypeSpecTest.java

```diff
@@ -53,6 +51,7 @@
   private static final String donutsPackage = "com.squareup.donuts";
 
   @Rule public final CompilationRule compilation = new CompilationRule();
+  @Rule public final ExpectedException expectedException = ExpectedException.none();
```

Comment:
Please don't use this Rule. It's strictly worse than the try/catch clauses it attempts to improve upon.